sass_processor/management/commands/compilescss.py

```diff
@@ -150,9 +150,17 @@ def handle(self, *args, **options):
 
             # find all Python files making up this project; They might invoke `sass_processor`
             for py_source in self.find_sources():
-                self.parse_source(py_source)
-                if self.verbosity > 0:
+                if self.verbosity > 1:
+                    self.stdout.write("Parsing file: %s" % py_source)
+                elif self.verbosity == 1:
                     self.stdout.write(".", ending="")
+                try:
+                    self.parse_source(py_source)
+                except SyntaxError as e:
+                    self.stderr.write("Syntax error encountered processing %s" % py_source)
+                    self.stderr.write("Aborting compilation")
```

Comment:
Reading the [Django docs about command errors](https://docs.djangoproject.com/en/2.0/howto/custom-management-commands/#command-exceptions), here we should raise a `CommandError`, or are there any specific reasons for this kind of exception?

Reply:
No specific reason for me to do this, it was just the first thing that came to mind.

Reading the docs, I agree that CommandError seems like a better choice. Let me twiddle..